pkgs/applications/misc/perkeep/default.nix

```diff
@@ -1,72 +1,41 @@
 { stdenv, lib, go, fetchzip, git, fetchpatch, fetchFromGitHub, fetchgit }:
 
-# When perkeep is updated all deps in the let block should be removed
-let
-  gopherjs = fetchFromGitHub {
-    owner = "gopherjs";
-    repo = "gopherjs";
-    # Rev matching https://github.com/perkeep/perkeep/commit/2e46fca5cc1179dbd90bec49fec3870e6eca6c45
-    rev = "b40cd48c38f9a18eb3db20d163bad78de12cf0b7";
-    sha256 = "0kniz8dg5bymb03qriizza1h3gpymf97vsgq9vd222282pdj0vyc";
-  };
-
-  gotool = fetchFromGitHub {
-    owner = "kisielk";
-    repo = "gotool";
-    rev = "80517062f582ea3340cd4baf70e86d539ae7d84d";
-    sha256 = "14af2pa0ssyp8bp2mvdw184s5wcysk6akil3wzxmr05wwy951iwn";
-  };
-
-  gcimporter15 = fetchgit {
-    url = "https://go.googlesource.com/tools";
-    rev = "f8f2f88271bf2c23f28a09d288d26507a9503c97";
-    sha256 = "1pchwizx1sdli59g8r0p4djfjkchcvh8msfpp3ibvz3xl250jh0n";
-  };
-
-in
 stdenv.mkDerivation rec {
```

Comment:
Can you use buildGoPackage here? Otherwise the result will depend on the go compiler at runtime.

Reply:
Sure, let me give that a try, I'll ping you once I addressed your comment. Thx @Mic92 for the review!